sourcegraph/sourcegraph
# Setup: Merge repo selection and embeddings (new external services API)

Based on https://github.com/sourcegraph/sourcegraph/pull/53838

Basically re-implement work from https://github.com/sourcegraph/sourcegraph/pull/53825 but on top of @fkling changes about external services in https://github.com/sourcegraph/sourcegraph/pull/53838

## Test plan
- Make sure that you can add no more than 10 repositories on the setup flow 
- You can add more repositories later on the setting page (local repositories tab)
- Check that embedings will be autogenerated/uploaded automatically for repositories from the setup flow.
